go.mod

```diff
@@ -4,53 +4,160 @@ go 1.19
 
 require (
 	github.com/dgrijalva/jwt-go v3.2.0+incompatible
-	github.com/gofrs/uuid v4.0.0+incompatible
 	github.com/lib/pq v1.10.2
 	github.com/pkg/errors v0.9.1
-	github.com/sirupsen/logrus v1.5.0
-	github.com/spf13/viper v1.14.0
+	github.com/sirupsen/logrus v1.9.0
+	github.com/stretchr/testify v1.8.1
+	github.com/testcontainers/testcontainers-go v0.16.0
 	goa.design/goa/v3 v3.3.1
-	golang.org/x/oauth2 v0.2.0
-	gopkg.in/square/go-jose.v2 v2.3.1
-	gopkg.in/yaml.v2 v2.4.0
 	gorm.io/gorm v1.24.1-0.20221019064659-5dd2bb482755
-
 )
 
 require (
+	github.com/AlecAivazis/survey/v2 v2.3.6 // indirect
+	github.com/Azure/go-ansiterm v0.0.0-20210617225240-d185dfc1b5a1 // indirect
+	github.com/Microsoft/go-winio v0.5.2 // indirect
+	github.com/beorn7/perks v1.0.1 // indirect
+	github.com/buger/goterm v1.0.4 // indirect
+	github.com/cenkalti/backoff/v4 v4.1.3 // indirect
+	github.com/cespare/xxhash/v2 v2.1.2 // indirect
+	github.com/compose-spec/compose-go v1.6.0 // indirect
+	github.com/containerd/console v1.0.3 // indirect
+	github.com/containerd/containerd v1.6.8 // indirect
+	github.com/containerd/continuity v0.3.0 // indirect
+	github.com/containerd/ttrpc v1.1.0 // indirect
+	github.com/containerd/typeurl v1.0.2 // indirect
+	github.com/davecgh/go-spew v1.1.1 // indirect
 	github.com/dimfeld/httptreemux/v5 v5.2.2 // indirect
-	github.com/fsnotify/fsnotify v1.6.0 // indirect
+	github.com/distribution/distribution/v3 v3.0.0-20220907155224-78b9c98c5c31 // indirect
+	github.com/docker/buildx v0.9.1 // indirect
+	github.com/docker/cli v20.10.19+incompatible // indirect
```

Comment:
I wonder what requires docker modules?

Reply:
This is a requirement for testcontainers.